webpack/assets/javascripts/react_app/components/bookmarks/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Dropdown, MenuItem } from 'react-bootstrap';
+import SearchModal from './SearchModal';
+import Bookmark from './Bookmark';
+import * as BookmarkActions from '../../redux/actions/bookmarks';
+import DocumentationUrl from '../common/DocumentationLink';
+
+class BookmarkContainer extends React.Component {
+  componentDidMount() {
+    const { url, controller, getBookmarks } = this.props;
+
+    getBookmarks(url, controller);
+  }
+
+  onClick() {
+    this.props.showModal ? this.props.modalClosed() : this.props.modalOpened();
```

Comment:
This will be fixed by not using a ternary and just using a regular if/else.

Reply:
Ok... but why is this syntax is wrong?

Reply:
Ternary operators are actually for assignment - not really for shorthand if/else (even though they're often used that way). They actually return a value and the linting rule says you're ignoring the return value (since it's not being assigned/used) which could be a logic error.